src/BlueSCSI.cpp

```diff
@@ -920,9 +920,6 @@ void writeDataLoop(uint32_t blocksize, const byte* srcptr)
   // Start the first bus cycle.
   FETCH_BSRR_DB();
   REQ_OFF_DB_SET(bsrr_val);
-#ifdef XCVR
-  TRANSCEIVER_IO_SET(vTR_DBP,TR_OUTPUT)
```

Comment:
Restoring this segment increases read speed to 1335/1383 on beige G3 (three benchmarks run on two different disk images, this is the average of results).

Despite making the code appear less efficient, speeds are increased with this segment.

Reply:
It will only be faster on SCSI-2 Macs (Quadras and later). If you take a look at the numbers I saw, the LC III+ read speed dropped from 1247 to 1137, about a 9% performance hit, so that alignment change is not a clear win. I've been deliberately tuning the performance to try and get good speeds across the board, rather than favoring any particular computers.